packages/xo-server-transport-nagios/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "xo-server-transport-nagios",
   "version": "0.1.0",
   "license": "ISC",
-  "description": "",
+  "description": "The Nagios transport plugin for XO-Server",
```

Comment:
As I said, description should provide some information, and avoid obvious info like *plugin*, *xo-server*, etc.

This one does not.

Reply:
My bad, I didn't see your comment.

Reply:
```suggestion
  "description": "Send backup runs statuses to Nagios",
```